UI/volume-control.cpp

```diff
@@ -751,14 +751,14 @@ void VolumeMeter::paintEvent(QPaintEvent *event)
 		tickPaintCache->fill(clearColor);
 
 		QPainter tickPainter(tickPaintCache);
-		paintTicks(tickPainter, 0, 0, tickPaintCacheSize.width(),
+		paintTicks(tickPainter, 6, 0, tickPaintCacheSize.width() - 6,
```

Comment:
Is this 6 instead of 5 as a slight nudge to get the ticks to line up with the start and end of the meter-bar?
I guess that is fine, it would solve the problem rather well.

Reply:
Yes exactly, I did a long session of trial and error with 0-10 starting Xs and print screen + gimp full zoom to check the aligned pixels. Those values were the best i've got.

Reply:
Yea my in-my-mind-calculation would say that 6 is exactly right.

Reply:
The original 5 would draw the -60 tick exactly one full pixel to the left of the meter.